conf/build.manifest

```diff
@@ -1,6 +1,7 @@
 ---
 code_name: "Bluefin"
 debian_release: "bullseye"
+identity_file_path_default: "~/.ssh/id_rsa"
```

Comment:
We'll have to expand the current users home directory or this won't work. i.e. `os.path.expanduser('~') + /.ssh/id_rsa`

Reply:
Oh I see you're doing this down below.